src/pages/login/login.tsx

```diff
@@ -90,6 +118,11 @@ export class Login extends React.Component<Props, State> {
                   <FormGroup label={t('login.password_label')}>
                     <TextInput
                       {...getTestProps(testIds.login.password_input)}
+                      {...(error &&
```

Comment:
This can be written as `isError={error && error.response && error.response.data.password}` if the typings ask for a boolean you can strictly cast it with `Boolean(error && error.response && error.response.data.password)` Same for the error above. It might be nice to move this up and assign them variable names something like
```jsx
const passwordError = error && error.response && error.response.data.password;
const usernameError = error && error.response && error.response.data.username;

<Alert>{usernameError || passwordError}</Alert>
<TextInput  isError={Boolean(usernameError)} />
<TextInput  isError={Boolean(passwordError)} />
```

That way you only need to dive into is once for each error.